Ladybird/AppKit/UI/Tab.mm

```diff
@@ -205,15 +207,19 @@ - (void)updateTabTitleAndFavicon
     [favicon_attribute addAttribute:NSForegroundColorAttributeName
                               value:[NSColor clearColor]
                               range:NSMakeRange(0, [favicon_attribute length])];
+    // Add a offset to make the icon vertical center, maybe have better solution
+    [favicon_attribute addAttribute:NSBaselineOffsetAttributeName
+                              value:@-2
+                              range:NSMakeRange(0, [favicon_attribute length])];
 
     // By default, the text attachment will be aligned to the bottom of the string. We have to manually
     // try to center it vertically.
-    // FIXME: Figure out a way to programmatically arrive at a good NSBaselineOffsetAttributeName. Using
-    //        half the distance between the font's line height and the height of the favicon produces a
-    //        value that results in the title being aligned too low still.
```

Comment:
Let's move this FIXME comment above the new `NSBaselineOffsetAttributeName` of -2 above, as we didn't really resolve it here, we just changed where we hard-code the offset.